doc/book/src/development/e2e/run-tests.md

```diff
@@ -1 +1,66 @@
-# Run the test
+# Run End-to-End tests
+
+To run the End-to-End tests, you need a QUAY repository and admin access to an OpenShift cluster where [KubeSaw][kubesaw] and [Konflux-Workspaces][konflux-workspaces] are running.
+
+## All in one script
+
+To easily setup the cluster you can refer to the [`./hack/demo.sh` script][hack-demo-script].
+This script will install [KubeSaw][kubesaw] and [Konflux-Workspaces][konflux-workspaces], and then execute e2e tests.
+
+## Step by step
+
+### Install dependencies
+
+You need to define some variables to use in the next steps.
+
+```sh
+# the tag to use for KubeSaw images build at step 1
+export IMAGE_TAG=e2e-test
+
+# the quay.io namespace to use in the next steps
+export QUAY_NAMESPACE=my-quay-namespace
+```
+
```

Comment:
Maybe call out the `IMAGE_BUILDER` env var?

Reply:
good idea, I'll do this a new PR

Reply:
#223